src/main/resources/config/init.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~  Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com)
+ ~
+ ~  WSO2 LLC. licenses this file to you under the Apache License,
+ ~  Version 2.0 (the "License"); you may not use this file except
+ ~  in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing,
+ ~  software distributed under the License is distributed on an
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+-->
+
+<template name="init" onError="fault" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter name="name" description="Connection name" />
+    <parameter name="server" description="The server URL" />
+    <parameter name="port" description="URL port"/>
+    <parameter name="bearerToken" description="Bearer Token type"/>
+    <parameter name="username" description="Username" />
+    <parameter name="password" description="Password"/>
+    <parameter name="securityToken" description="Security Token"/>
+    <parameter name="headers" description="Headers" />
+    <parameter name="tlsEnabled" description="TLS channel" />
+    <parameter name="blocking" description="Whether to use blocking transport"/>
+    <sequence>
+    <class name="org.wso2.integration.salesforcepubsub.GRPCChannelBuilder">
+        <property name="name" expression="$func:name" />
```

Comment:
It is not recommended to pass parameters to class mediator. It is better to use lookup functions inside the class mediator and retrieve the parameters. Similar to https://github.com/wso2-extensions/esb-connector-email/blob/master/src/main/java/org/wso2/carbon/connector/operations/EmailSend.java#L84